packages/office-ui-fabric-react/src/components/Rating/docs/new/RatingBestPractices.md

```diff
@@ -0,0 +1,9 @@
+### Layout
+
+- Make it clear which product the rating pertains to by making sure the layout and grouping are clear when several products are on the page.
+- Don’t use the rating component for data that has a continuous range, such as the brightness of a photo. Instead, use a slider.
+
+### Content
+
+- Use a five-star rating system.
+- Label shouldn’t include a color and should follow sentence casing.
```

Comment:
I'm confused why the color part is worth mentioning--when might someone think they should put a color in a rating label?